src/pow.cpp

```diff
@@ -428,6 +432,8 @@ RCP<const Basic> pow_expand(const RCP<const Pow> &self)
         }
     }
     RCP<const Basic> result = Add::from_dict(add_overall_coeff, std::move(rd));
+    RCP<const Number> i = integer(-1);
+    if(negative_pow) result = rcp(new Pow(result, i));
```

Comment:
I'm getting a weird error on trying this
```
>>> (1/((x + y)**2 - x)).expand()
```
https://gist.github.com/shivamvats/87f24b310da644a43bab